src/main/java/com/iota/iri/service/dto/GetTransactionsToApproveResponse.java

```diff
@@ -14,10 +23,20 @@ public static AbstractResponse create(Hash trunkTransactionToApprove, Hash branc
 		return res;
 	}
 	
+    /**
+     * The branch transaction
+     *
+     * @return The branch transaction.
+     */
```

Comment:
extraneous docs

Reply:
apparently not...
Didn't understand how the doclet worked when I made the comment